prometheus/config/args/clean.sls

```diff
@@ -45,10 +45,15 @@ prometheus-config-args-{{ name }}-all:
       - service: prometheus-service-clean-{{ name }}-service-dead
 
             {%- elif grains.os_family != 'FreeBSD' %}
+              {%- if grains.os_family == 'Debian' %}
+                {%- set config_file_name = '{}/{}'.format(prometheus.dir.args, name) %}
+              {%- else %}
+                {%- set config_file_name = '{}/{}.sh'.format(prometheus.dir.args, name) %}
+              {%- endif %}
```

Comment:
This won't fly with Ubuntu 16.04 (and I suppose more recent versions too). Node exporter is packaged as `prometheus-node-exporter` there and wants `/etc/default/prometheus-node-exporter`.